common/djangoapps/student/management/commands/anonymized_id_mapping.py

```diff
@@ -23,20 +23,17 @@ class Command(BaseCommand):
     # TODO: revisit now that rake has been deprecated
```

Comment:
Nit: Is this true? Worth deleting?

Reply:
I'm happy to delete it. The only other thing I can see people wanted to add args for is the path to the output file, which folks can add if it bothers them.